### PR TITLE
MAISTRA-1216 Don't log preemption warning when using Multus

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: deployments/kubernetes/install/scripts/install-cni.sh [hunk 1]
```diff
@@ -50,15 +50,17 @@ function find_cni_conf_file() {
...
-  cfgfile_nm=$(find_cni_conf_file)
-  if [[ "${cfgfile_nm}" != "${CNI_CONF_NAME}" ]]; then
-    if [[ "${CNI_CONF_NAME_OVERRIDE}" != "" ]]; then
-       # Install was run with overridden cni config file so don't error out on the preempt check.
-       # Likely the only use for this is testing this script.
-       echo "WARNING: Configured CNI config file \"${MOUNTED_CNI_NET_DIR}/${CNI_CONF_NAME}\" preempted by \"$cfgfile_nm\"."
-    else
-       echo "ERROR: CNI config file \"${MOUNTED_CNI_NET_DIR}/${CNI_CONF_NAME}\" preempted by \"$cfgfile_nm\"."
-       exit 1
+  if [ "${CHAINED_CNI_PLUGIN}" != "true" ]; then
```
Comment (jwendell) [inline deployments/kubernetes/install/scripts/install-cni.sh]: I think you mean `==`.

Anyway, I think it's better check if it's `!= true` and do an early exit. It's a less intrusive change.

Comment (luksa) [reply deployments/kubernetes/install/scripts/install-cni.sh]: OMG, you're right. Thanks!!!

Yeah, I usually use early exists, but in this case, only the first part of the function needs to be skipped.

**Review by jwendell — APPROVED** [review]
Submit upstream as well please

**Review by dgn — APPROVED** [review]

**Comment by luksa** [comment]
> Submit upstream as well please

Done.

**Pull request merged**